Sources/GoodNetworking/Session/Logger/LoggingEventMonitor.swift

```diff
@@ -10,6 +10,10 @@ import Alamofire
 
 public class LoggingEventMonitor: EventMonitor {
 
+    public static var verbose: Bool = true
+    public static var prettyPrinted: Bool = true
+    public static var dataLimit: Int = 100_000
```

Comment:
co znamena `dataLimit = 100_000`? citam to o 2 roky neskor 😄 

Radsej by som to pomenoval `maxLoggingDataSize`/`maxVerboseLogSizeBytes`/... Spravny naming by vzdy mal pomenovat tu premennu podla toho co realne je a aku ma jednotku (cize maximalna velkost, ktoru ked request body prekroci, uz sa nebude logovat, v bajtoch).

Reply:
suhlasim, chce to lepsi nazov 😄